.gitignore

```diff
@@ -21,5 +21,6 @@ __pycache__
 .DS_Store
 \#*#
 .#*
+.cache
```

Comment:
Should that also be in your system global `.gitignore` if you feel the need to add it ?

Reply:
Perhaps, but it's in the common Python one --https://github.com/github/gitignore/blob/master/Python.gitignore#L43 -- and ignored in some other Python projects I just checked. And it showed up after running the tests for this project (`iptest --coverage xml `), so it can show up for others too.

Reply:
Yeah, I'm not sure exactly what creates it, but I see it quite often when running tests, so I'm fine with adding it to gitignore.